src/components/views/Navbar/Sections/RightMenu.jsx

```diff
@@ -1,24 +1,64 @@
 import React, { useState } from 'react';
-import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link, useNavigate } from 'react-router-dom';
+import { useCookies } from 'react-cookie';
+import { isEmpty } from 'lodash';
+import { logoutUser } from '../../../../_actions/user_action';
 import { Menu } from 'antd';
-import { EditOutlined, LoginOutlined, LogoutOutlined } from '@ant-design/icons';
+import {
+	EditOutlined,
+	LoginOutlined,
+	LogoutOutlined,
+	IdcardOutlined,
+} from '@ant-design/icons';
 
 function RightMenu() {
-	const [currentPageKey, setCurrentPageKey] = useState('');
+	const navigate = useNavigate();
+	const dispatch = useDispatch();
 
+	const [currentPageKey, setCurrentPageKey] = useState('');
+	const { loginSuccess } = useSelector((state) => state.user);
+	const [, , removeCookie] = useCookies(['TID']); // const [cookie, setCookie, removeCookie] = useCookies(['TID']);
+	console.log(loginSuccess);
 	const handleClick = (e) => {
 		setCurrentPageKey(e.key);
 	};
 
+	const handleLogout = () => {
+		removeCookie('TID');
```

Comment:
```suggestion
		removeCookie('TID');
```

해당 부분은 action 함수안으로 이동해도 좋을것 같아요!

Reply:
반영했습니당!